src/config/ConfigManager.cpp

```diff
@@ -349,6 +349,7 @@ CConfigManager::CConfigManager() {
     m_pConfig->addConfigValue("misc:background_color", Hyprlang::INT{0xff111111});
     m_pConfig->addConfigValue("misc:new_window_takes_over_fullscreen", Hyprlang::INT{0});
     m_pConfig->addConfigValue("misc:enable_hyprcursor", Hyprlang::INT{1});
+    m_pConfig->addConfigValue("misc:animate_across_monitors", Hyprlang::INT{1});
```

Comment:
well now you need a wiki MR. My question is, why is this an option? I am not in favor of this being an option

Reply:
Ref: https://discord.com/channels/961691461554950145/967398886530515067/1220460557753122928
if you have different scaling/DPI ratios on different monitors it might look weird

Reply:
it already will if you have a window partially on one or the other. (floating)

Reply:
That is indeed true. I couldn't think of a better way to address that except for adding that config option, but I will remove the config for now

Reply:
changed